_ml-commons-plugin/cluster-settings.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: ML Commons cluster settings
+has_children: false
+nav_order: 10
+---
+
+# ML Commons cluster settings
+
+This page provides an overview of `opensearch.yml` settings that can be configured for the ML commons plugin.
+
+| **Setting Name** | **Default Value** | **Value Range** | **Description** |
+|:---:|:---:|:---:|:---:|
+| `plugins.ml_commons.only_run_on_ml_node` | false | true or false | If `true`, ML Commons task and models run machine learning (ML) tasks on ML nodes only. If `false`, tasks and models run on ML nodes first. If no ML nodes exist, tasks and model run on data nodes. Don't set as "false" on production cluster. |
+| `plugins.ml_commons.task_dispatch_policy` | round_robin |  round_robin or least_load | Default is `round_robin`. `round_  robin` dispatches ML tasks to ML nodes using round robin routing. `least_load` gathers all an ML nodes' runtime information, like JVM heap memory usage and running tasks, then dispatches tasks to the ML node with the least load. |
+| `plugins.ml_commons.sync_up_job_interval_in_seconds` | 10 | From 0 (included) to 86400 (included) | If set as 0, ML Commons immediately stops any syncup jobs |
+| `plugins.ml_commons.monitoring_request_coun`t | 100 | From 0 (included) to 10_000_000 (included) | Controls how many predict requests are monitored on one node. If set to `0`, ML Commons cleans up monitoring requests and won't monitor predict requests. |
```

Comment:
Are there supposed to be underscores between groups in `10_000_000`? That isn't very intuitive. Would you specify for a value of 9001?  Would it be 9001? 9,001? 9_001?

Reply:
This format is a programming language format. I suggest changing it to human readable 10,000,000.

Reply:
@ylwu-amzn: Do you have a preference between the top of the range appearing as `10,000,000` or `10_000_000`?

Reply:
Yes, `10_000_000` is programming format. I'm fine to change to `10,000,000`